src/config/session.c

```diff
@@ -20,7 +20,10 @@ static const char *const env_vars[] = {
 	"DISPLAY",
 	"WAYLAND_DISPLAY",
 	"XDG_CURRENT_DESKTOP",
-	NULL,
+	"XCURSOR_SIZE",
+	"XCURSOR_THEME",
+	"XDG_SESSION_TYPE",
+	"LABWC_PID"
```

Comment:
This list needs a NULL terminator to signal its end:
```suggestion
	"LABWC_PID",
	NULL
```